vizro-core/src/vizro/static/css/layout.css

```
@@ -175,3 +176,24 @@ div.dashboard_container .custom-tooltip {
 #page-container div[hidden] {
   display: none;
 }
+
+.collapsable-div {
+  background-color: var(--surfaces-bg-02);
+  height: 100%;
+  padding-top: 44px;
+  width: 12px;
+  padding-right: 12px;
+}
+
+#collapse_icon.material-symbols-outlined {
+  z-index: 1;
+  position: absolute;
+  background-color: var(--surfaces-bg-card);
+  border-radius: 50%;
+  cursor: pointer;
+  color: var(--text-secondary);
+}
+
+#collapse_icon.material-symbols-outlined:hover {
+  color: var(--text-active);
+}
```

This is the change you would need to position the tooltip correctly without needing the inline style and an outer div

```suggestion
}

.collapse-button-tooltip {
  position: relative;
  top: 44px;
  left: -12px;
  height: 24px;
  width: 0 !important;
}
```